### ISSUE TITLE
Evaluación ambigua 

### ISSUE THREAD

**Issue by fou-foo** [RodrigoZepeda/CapacidadHospitalariaMX#3]
"Capacidad predictiva. Este repositorio es para construir modelos que nos sirvan para saber cómo se va ser la ocupación hospitalaria en el país. No necesitamos que el modelo explique, sólo que prediga."
Lo anterior no define un método de selección de modelos o bondad de ajuste, además el código no ejemplifica ninguna prueba efectuada hacerca de la evaluación del resultado. Bajo este enfoque todo es subjetivo.

**Comment by RodrigoZepeda** [comment]
No hemos agregado las pruebas que tenemos danos al viernes próximo para subirlas y poder reescribir eso. En breve lo único que checamos es la verdadera probabilidad de los intervalos (conteo de cuántos caen en el ic / total )

**Comment by fou-foo** [comment]
'verdadera probabilidad', seguimos a la espera

**Comment by RodrigoZepeda** [comment]
Sí, perdón hemos estado corriendo. @valperez ¿podrías subir el código de evaluación?. 

Lo que intenté decir es que para cada uno de los periodos seleccionados $t$, para cada momento $s$ ($0 < s < 90$) si se observó $x_{t+s}$ y el modelo al $(1 - \alpha) \times 100\%$ (credibilidad) generó unos intervalos $C_s = [l_{s+t}, u_{t+s}]$ estimamos la "verdadera probabilidad" de los intervalos en el periodo $[t,t+90]$ mediante:
$$
\hat{p}_t = \sum_s \dfrac{\mathbb{I}_{C_s}(x_{t+s}) }{90}
$$

Ahora, evaluar así tiene sus desventajas (digamos el IC $[0,1]$ siempre va a funcionar perfecto pero no es la idea) y no he pensado a profundidad cómo integrar la amplitud del IC a la función de pérdida ¿alguna idea @fou-foo ? También si consideras una mejor manera de evaluar estamos abiertos.
